From d2e77ba768c7fb2c72e7d3e78d1118bf523ddfa2 Mon Sep 17 00:00:00 2001
From: Patrice Vilchez <patrice.vilchez@atmel.com>
Date: Wed, 16 Jan 2013 17:06:34 +0100
Subject: [PATCH 298/319] drivers/video/atmel_hlcdfb.c: Remove bad default
 configuration

By default the LCD controller is configured to "launch" pixel data and control signals on the
falling edge of the pixel clock. We have removed this default configuration.

Signed-off-by: Patrice Vilchez <patrice.vilchez@atmel.com>
---
 drivers/video/atmel_hlcdfb.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/video/atmel_hlcdfb.c b/drivers/video/atmel_hlcdfb.c
index dd9dcee..c2ef5d7 100644
--- a/drivers/video/atmel_hlcdfb.c
+++ b/drivers/video/atmel_hlcdfb.c
@@ -261,7 +261,7 @@ static int atmel_hlcdfb_setup_core_base(struct fb_info *info)
 
 	if (value < 1) {
 		dev_notice(info->device, "using system clock as pixel clock\n");
-		value = LCDC_LCDCFG0_CLKPOL | LCDC_LCDCFG0_CLKPWMSEL | LCDC_LCDCFG0_CGDISBASE;
+		value = LCDC_LCDCFG0_CLKPWMSEL | LCDC_LCDCFG0_CGDISBASE;
 		lcdc_writel(sinfo, ATMEL_LCDC_LCDCFG0, value);
 	} else {
 		info->var.pixclock = KHZ2PICOS(clk_value_khz / value);
@@ -271,7 +271,6 @@ static int atmel_hlcdfb_setup_core_base(struct fb_info *info)
 		dev_dbg(info->device, "  * programming CLKDIV = 0x%08lx\n",
 					value);
 		value = (value << LCDC_LCDCFG0_CLKDIV_OFFSET)
-			| LCDC_LCDCFG0_CLKPOL
 			| LCDC_LCDCFG0_CGDISBASE;
 		lcdc_writel(sinfo, ATMEL_LCDC_LCDCFG0, value);
 	}
-- 
1.7.10.4

